index.html

```diff
@@ -49,5 +49,7 @@
     <script>
       navigator.serviceWorker.register("/pwabuilder-sw.js");
     </script>
+
+    <script src="https://unpkg.com/ionicons@5.2.3/dist/ionicons.js"></script>
```

Comment:
Even though it's small, I wonder, can we make this async?

Reply:
Much agreed, fixed in the commit below (: